services/src/datasets/external/netcdfcf/overviews.rs

```diff
@@ -435,108 +620,185 @@ fn index_subdataset<C: TaskContext>(
         conversion.dataset_out_base.display()
     );
 
-    let subdataset = gdal_open_dataset_ex(
-        Path::new(&conversion.dataset_in),
-        DatasetOptions {
-            open_flags: GdalOpenFlags::GDAL_OF_READONLY | GdalOpenFlags::GDAL_OF_MULTIDIM_RASTER,
-            allowed_drivers: Some(&["netCDF"]),
-            open_options: None,
-            sibling_files: None,
-        },
-    )
-    .boxed_context(error::CannotOpenNetCdfSubdataset)?;
+    let mut subdataset = gdal_netcdf_open(None, Path::new(&conversion.dataset_in))
+        .boxed_context(error::CannotOpenNetCdfSubdataset)?;
 
-    let raster_creation_options = CogRasterCreationOptions::new(resampling_method)?;
-    let raster_creation_options = raster_creation_options.options();
+    let raster_creation_options = Arc::new(CogRasterCreationOptions::new(resampling_method)?);
 
     debug!(
         "Overview creation GDAL options: {:?}",
         &raster_creation_options
     );
 
-    let time_steps = time_coverage.time_steps()?;
-
-    let (mut value_min, mut value_max) = (f64::INFINITY, -f64::INFINITY);
-
     for entity in 0..conversion.number_of_entities {
         emit_subtask_status(
             conversion_index,
             number_of_conversions,
             entity as u32,
             conversion.number_of_entities as u32,
             task_context,
-        );
+        )
+        .await;
+
+        let loading_info_stmt = db_transaction
+            .prepare_typed(
+                r#"
+            INSERT INTO loading_infos (
+                file_name,
+                group_names,
+                entity_id,
+                meta_data
+            ) VALUES (
+                $1,
+                $2 :: TEXT[],
+                $3,
+                $4 :: "GdalMetaDataList"
```

Comment:
just put NULL here if you never set it?

Reply:
I don't understand what you mean here?

Reply:
Ah… no… we will set this.
But I can't type it here since I don't have a type like `tokio_postgres::types::Type::INT8`. I have no clue what the OID of this type is.
But the value is set later on… just without pre-typing here.